server/database/queries.js

```diff
@@ -102,8 +100,24 @@ const getListsAndCardsForBoard = (board) => {
         .orderBy('list_id', 'asc')
         .orderBy('order', 'asc')
         .then(cards => {
-          board.cards = cards
-          return board
+          const cardIds = cards.map( card => card.id )
+          return knex.table('card_labels')
+            .select('*')
+            .whereIn( 'card_id', cardIds )
+            .join('labels','card_labels.label_id','labels.id')
+            .then( cardLabels => {
+              cards = cards.map( card => {
```

Comment:
you dont need to map here you just need a forEach that adds the `label_ids` property to each card